amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/common/fm/FMSettingsFlat.java

```diff
@@ -0,0 +1,28 @@
+package org.digijava.kernel.ampapi.endpoints.common.fm;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import io.swagger.annotations.ApiModelProperty;
+
+public class FMSettingsFlat {
+    
+    @ApiModelProperty(value = "list of FM settings",
+            example = "### Use cases\n"
+                    + "1. Detail flat & fully enabled paths => /Activity Form/Organiation/Donor Organization\n"
```

Comment:
Since this is for flat EP branch only, we no longer need to clarify the `Detail flat` text. Or you can move up to the value `list of FM settings in flat format`